### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by matee911**

### CODE REVIEW

**Comment by ekamil** [comment]
Hi,
Please fix the commit message to meet our guidelines

File: README.md [hunk 1]
```diff
@@ -7,3 +7,18 @@ Every Rabbit(MQ) needs a good Clay Pipe.
...
+
+## Dev guide
+
+### Prerequisites
+
+* [.NET Core](https://www.microsoft.com/net/core)
+* [RabbitMQ](http://www.rabbitmq.com/)
+* [PostgreSQL](https://www.postgresql.org/)
+
+### ...
```
Comment (ekamil) [inline README.md]: Instead of "…": Running 
:)

**Comment by matee911** [comment]
@ekamil where are our guidelines?

**Comment by ekamil** [comment]
From our git overlords: https://git-scm.com/book/ch5-2.html#Commit-Guidelines

**Pull request merged**